src/containers/Accounts/AccountHeader/BalanceSelector/BalanceSelector.tsx

```diff
@@ -28,6 +28,7 @@ export const BalanceSelector = ({
         return (
           <BalanceSelectorItem
             currency={currency}
+            key={currency}
```

Comment:
Woops.  I forgot to add this.

Reply:
The console yelled at me every time I opened an account page, and it was a quick fix